src/components/Book.js

```diff
@@ -11,10 +11,25 @@ const Book = (props) => {
     dispatch(removebook(book, apikey));
   };
   return (
-    <div>
-      {book.title}
-      {book.author}
-      <Button id={book.id} handleClickEvent={() => removeBook(book)} name="Remove" />
+    <div className="display-panel">
+      <div>
+        <div className="cat-name">{book.category}</div>
+        <div className="book-title">{book.title}</div>
+        <div className="auth-name">{book.author}</div>
+        <ul className="book-tags">
+          <li><Button className="book-tags-button" id={book.id} name="Comments" /></li>
+          <li><Button className="book-tags-button" id={book.id} handleClickEvent={() => removeBook(book)} name="Remove" /></li>
+          <li><Button className="book-tags-button" id={book.id} name="Edit" /></li>
+        </ul>
+      </div>
+      <div className="progress">
+        <div className="circular-progress-container">
+          <div className=".circular-progress" />
+        </div>
+        <div>&nbsp;</div>
+        <div>&nbsp;</div>
+        <div>&nbsp;</div>
```

Comment:
- [ ] Kindly try to match your design to look like the given design as much as possible. The circular progress bar is missing from your design, so is the progress information to the far left of the original design, such as the chapter e.t.c., kindly add it.

Given design
![image](https://user-images.githubusercontent.com/81932478/201429878-3ba19cc6-3fb0-4d0b-b4f5-c33aa237c517.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/201430015-25e576e9-2315-4cef-beb9-33810e74d8b2.png)